autoload/vsnip.vim

```diff
@@ -98,7 +98,8 @@ endfunction
 " get_context.
 "
 function! vsnip#get_context() abort
-  let l:before_text = getline('.')[0 : col('.') - 2]
+  let l:offset = mode() ==# 's' ? 1 : 2
```

Comment:
Could you change with `mode()[0] ==# 'i' ? 2 : 1` instead.

Reply:
Done 👍